src/includes/enriching-events/scopes/with-scope/dart.mdx

```diff
@@ -0,0 +1,22 @@
+<Note>
+
+This feature is available as a Beta. Features available in Beta are still in-progress and may have bugs. We recognize the irony.
+
+This is available in 5.1.0.beta.1 version of the Sentry Dart/Flutter SDK.
+
```

Comment:
```suggestion
```

as its the the beta.1, we don't need to mention that.\, it'd make sense if its available under beta.2, 3 etc